src/javascripts/Services/gridService.js

```diff
@@ -0,0 +1,90 @@
+// Base view to show nodes and columns. It's like a tree view but with no depth. 
+// Utilized by all views. Functions for specialized views (e.g. treeView/recordView)
```

Comment:
oh, interesting.  I wasn't expecting this, but it may work out.  will have to pay close attention then.

Reply:
Ultimately, I'm not sure if this is even a good idea.  While reducing code repetition is a good thing this may be making things too complicated.  I'll finish my review and come back to this, but I'm tempted to say I'd prefer to have some code repetition and make the grid component code more independent of the tree code.